staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -324,9 +324,12 @@ func RESTClientFor(config *Config) (*RESTClient, error) {
 	var httpClient *http.Client
 	if transport != http.DefaultTransport {
 		httpClient = &http.Client{Transport: transport}
-		if config.Timeout > 0 {
-			httpClient.Timeout = config.Timeout
```

Comment:
With this removed, how do I create a rest.Config that has a client-side, context based timeout on every request?  I was fairly sure this was exactly the right spot to set up such a timeout.

Reply:
>  how do I create a rest.Config that has a client-side, context based timeout on every request?

In exactly the same way as before. This behavior hasn't changed.